model/registry/http.yaml

```diff
@@ -6,11 +6,11 @@ groups:
     attributes:
       - id: request.body.size
         type: int
-        brief: >
-          The size of the request payload body in bytes. This is the number of bytes transferred excluding headers and
-          is often, but not always, present as the [Content-Length](https://www.rfc-editor.org/rfc/rfc9110.html#field.content-length)
-          header. For requests using transport encoding, this should be the compressed size.
+        brief: Size of the HTTP request body.
         examples: 3495
+        stability: experimental # this should not be marked stable with other HTTP attributes
```

Comment:
carried over from #460 just to make sure it doesn't get lost, will rebase this PR after #460 is merged